wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -1052,6 +1052,7 @@ impl super::Adapter {
                 spv::Capability::ImageQuery,
                 spv::Capability::DerivativeControl,
                 spv::Capability::SampledCubeArray,
+                spv::Capability::MultiView,
```

Comment:
multi-view needs to be an optional feature exposed by the adapter and then activated by the user on device request.
The logic here, therefore, should be to only include this multi-view capability if the feature is activated.